acl/src/main/java/org/apache/rocketmq/acl/common/AclUtils.java

```diff
@@ -120,7 +120,7 @@ public static void verify(String netaddress, int index) {
         }
     }
 
-    public static String[] getAddreeStrArray(String netaddress, String partialAddress) {
+    public static String[] getAddressStrArray(String netaddress, String partialAddress) {
```

Comment:
IMHO, getAddressStrArray => getAddresses() would be better.

Reply:
Okay,I will fix it.

Reply:
Thx,I have Done.